contrib/tools/workflowcheck/determinism/checker.go

```diff
@@ -381,6 +366,41 @@ func (c *collector) collectFuncInfo(fn *types.Func, decl *ast.FuncDecl) {
 	})
 }
 
+func (c *collector) checkRangeType(rangeType types.Type, n ast.Node, fn *types.Func) Reason {
+L: // Unwrap named or generic type
```

Comment:
`L:`?

Reply:
Should I change it to `Loop:`?

Reply:
Originally I was confused why you added a  label at all. I see  you changed the if-else to a select so it is needed now.

Bit of a nitpick, but I'd prefer the label be `Loop`

Reply:
fixed